src/components/Jobs/components/PreviewsJobs/PreviewsJobs.js

```diff
@@ -1,39 +1,38 @@
 import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
-import get from 'lodash/get';
+import { FormattedMessage } from 'react-intl';
+import {
+  get,
+  filter,
+} from 'lodash';
 
 import JobsList from '../JobsList';
 import sortPreviewJobs from './sortPreviewJobs';
 import { DataFetcherContext } from '../../../DataFetcher/DataFetcherContext';
+import {
+  READY_FOR_PREVIEW,
+  PREPARING_FOR_PREVIEW,
+} from '../../jobStatuses';
 
 class PreviewsJobs extends PureComponent {
-  static propTypes = {
-    checkDateIsToday: PropTypes.func.isRequired,
-    noJobsMessage: PropTypes.node,
-  };
+  static contextType = DataFetcherContext;
 
-  static defaultProps = {
-    noJobsMessage: '',
-  };
+  prepareJobsData() {
+    let jobs = get(this.context, ['jobs', 'itemsObject', 'jobExecutions'], []);
```

Comment:
I think it is okay to chain array built-in `.filter` method right here and define the variable as `const`.